ApiFirst/ApiFirst/Controllers/CustomersController.cs

```diff
@@ -2,8 +2,9 @@
 using AppServices.Interfaces;
 using Microsoft.AspNetCore.Mvc;
 using System;
+using System.Threading.Tasks;
 
```

Comment:
Essas referências são mesmo necessárias?
```suggestion
using Microsoft.AspNetCore.Mvc;

```